Casks/z/zen-browser.rb

```diff
@@ -0,0 +1,27 @@
+cask "zen-browser" do
+  arch arm: "aarch64", intel: "x64"
+
+  version "1.0.0-a.39"
+  sha256 arm:   "9e637193ef722ffab3491a7fe9a2a1725c562d8ee21bea38df08dc5d90067cb0",
+         intel: "19df7b246926537808f1d78534e305893bb0d4505e0c5a303a58efeaa930a3b4"
+
+  url "https://github.com/zen-browser/desktop/releases/download/#{version}/zen.macos-#{arch}.dmg",
+      verified: "github.com/zen-browser/desktop/"
+  name "Zen Browser"
+  desc "Customizable Gecko-based browser"
```

Comment:
We use British English, and generally don't include specific features here.

```suggestion
  desc "Gecko based web browser"
```